### ISSUE TITLE
Change of command name

### ISSUE THREAD

**Issue by madsbuch** [graphql-editor/graphql-editor#601]
Hi

This might be more of a question. I have just experienced that our entire infrastructure around generating graphQl has broken down. it seems like the command to generate typings for Zeus has changed from `graphql-zeus` to `zeus`.

It seems like everything is fixed by just renaming the command.

What baffles me is that this has be done in a patch update. Ie. from 5.3.2 to 5.3.3.

Is this the case?

**Comment by madsbuch** [comment]
It can be that it has been my update of bun to bun 1.1.0 that has introduced the error. I am just super curios.

**Comment by aexol** [comment]
Yeah we did not update the namings. However if you use npx to run zeus without installation then you run:
`npx graphql-zeus` as this is the package name. If you have it locally you can also use `zeus` binary. so zeus is just an alias
